exercises/takudzwa_makoni/ex02_oo_basics/src/isoTriangle.h

```diff
@@ -0,0 +1,31 @@
+#ifndef ISOTRIANGLE_H
+#define ISOTRIANGLE_H
+
+class isoTriangle : public shape { // isosceles triangle class
+
+private:
+
+	
+	const std::string name;
```

Comment:
Consider making shape names static - they do not change from instance to instance.